src/CommonDBTM.php

```diff
@@ -2060,11 +2060,11 @@ public function prepareInputForUpdate($input)
     /**
      * Actions done after the UPDATE of the item in the database
      *
-     * @param boolean $history store changes history ? (default true)
+     * @param bool $history store changes history ? (default true)
      *
      * @return void
      **/
-    public function post_updateItem($history = true)
+    public function post_updateItem(bool $history = true): void
```

Comment:
Changing the signature of a method that is expected to be overriden requires to update the signature in every plugin to prevent this chang to trigger fatal errors. I consider it is too late for this change, we have to wait for GLPI 12.0 now.

```
$ php -r 'class CommonDBTM { public function post_updateItem(bool $history = true): void {} } class PluginTestItemtype extends CommonDBTM { public function post_updateItem($history = true) {} }'

PHP Fatal error:  Declaration of PluginTestItemtype::post_updateItem($history = true) must be compatible with CommonDBTM::post_updateItem(bool $history = true): void in Command line code on line 1
```